ihatemoney/history.py

```diff
@@ -38,7 +38,10 @@ def history_sort_key(history_item_dict):
 
 def describe_version(version_obj):
     """Use the base model str() function to describe a version object"""
-    return parent_class(type(version_obj)).__str__(version_obj)
+    if version_obj is None:
```

Comment:
```suggestion
    if not version_obj:
```

Reply:
It's a few characters shorter but less clear I feel. Anyway, I don't mind a lot...

Reply:
Let me explain why I added this comment (sorry if this felt a bit short). This is not testing the exact same thing, so it's not only a matter of preference.

Given the fact that we're not using types in the project (we could), testing for Falsey values allows to consider both `None` and `False`. This is a general thing I try to do in these situations. Here it doesn't really matter to be honest, but that's my natural tendency.

Feel free to update your code or not, and we can merge :-)

Reply:
Interesting, thanks.